### DISCUSSION TITLE
Need to have Link targets _blank for Tina Markdown

### DISCUSSION THREAD

**Discussion by annuai** [tinacms/tinacms#3501] [Help]
All links in a blog post opens by default in the same page. I need to make them open in a new page. How do I do that?

**Comment by jeffsee55** [comment]
If _all_ links need to open to a new page you can just do that with your `<a>` tag in the `<TinaMarkdown>` component:

```ts
<TinaMarkdown content={data.content} components={{a: props => <a {...props} target="_blank" />}} />
```

**Reply by BeatrisCicotosto** [reply]
Olá,
Poderia por favor mostrar como ficou no seu esquema?
